tools/site_admin/manage_special_days.php

```diff
@@ -417,7 +403,7 @@ public function _get_status_cell($status, $class = '')
 
 // ----------------------------------------------------------------------------
 
-function make_link($url, $label)
+function make_link(string $url, string $label): string
```

Comment:
Because `info_url` can be null, both of these need to accept null too (see line 224).

Reply:
Indeed, thanks. Fixed this in the next commit + a few extra potential issues caught by PHPStan level 9.

I am bit concerned about this not being caught by PHPStan in the first place so will look into why tomorrow.